Gruntfile.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(grunt) {
             " *  <%= pkg.homepage %>\n" +
             " *\n" +
             " *  Made by <%= pkg.author.name %>\n" +
-            " *  Under <%= pkg.licenses[0].type %> License (<%= pkg.licenses[0].url %>)\n" +
+            " *  Under <%= pkg.license %> License\n" +
```

Comment:
Updated template to reflect changes in commit https://github.com/danieltartarottisobrosa/jquery-maskmoney/commit/df01b1b6c5ebbf610f22f3b17aad5baf143463c2